ssh/ssh.go

```diff
@@ -90,9 +92,24 @@ func RemoteSSHCommand(user string, privateKey string, host string, port string,
 		}
 	}
 
-	log.Info("Running command %s", cmd)
-	err = session.Run(cmd)
-	if err != nil {
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
+
+	var runnerr error = nil;
+	// Run in a go routine so that we can wait for it to finish or a OS signal to get raised.
+	go func() {
+		log.Info("Running command %s", cmd)
+		runnerr = session.Run(cmd)
+		signals <- syscall.SIGCHLD
+	}()
+
+	signal := <- signals
+	if runnerr != nil {
+		if signal!= syscall.SIGCHLD {
```

Comment:
so when the run command finishes I want the guy which is waiting for a signal to unblock.  I just picked a signal that was not being used and sent that.